projects/packages/forms/src/blocks/contact-form/variation-picker.js

```diff
@@ -55,7 +55,7 @@ export default function VariationPicker( { blockName, setAttributes, clientId, c
 					'Start by selecting one of these templates, or browse patterns.',
 					'jetpack-forms'
 				) }
-				variations={ variations.filter( v => ! v.hiddenFromPicker ) }
+				variations={ ( variations || [] ).filter( v => ! v.hiddenFromPicker ) }
```

Comment:
What's this change for?

Reply:
Completely unrelated, sorry it slipped through. Seems legit though :)